styles.css

```diff
@@ -61,20 +72,27 @@ body {
 }
 
 .card-4 {
+  display: flex;
+  align-items: center;
   border: 1px solid #4d7298;
   border-radius: 10px;
   width: 500px;
 }
 
 .card-text-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
`justify-content: space-between` isn't doing anything because you are then aligning flex items "manually" (using the `margin-top`)